extensions/session-time-counter/.eslintrc.js

```diff
@@ -0,0 +1,20 @@
+/**@type {import('eslint').Linter.Config} */
+// eslint-disable-next-line no-undef
```

Comment:
Is this needed?  Seems odd in the eslint config itself.

Reply:
While I was working on this PR, I have used their [official vscode repo](https://github.com/microsoft/vscode-extension-samples) as a source on how to do this. They have nice documentation and samples.

This particular lines in eslint config were from the base files of their hello world sample in [here](https://github.com/microsoft/vscode-extension-samples/blob/f7c3737b6e116c12e896c71dfe8e59ff3e8b9f3d/helloworld-sample/.eslintrc.js#L1).